### PR TITLE
Added pop column

### PR DESCRIPTION
**Pull request by khoroshevskyi**

### CODE REVIEW

File: pepdbagent/modules/project.py [hunk 1]
```diff
@@ -215,6 +222,7 @@ def create(
         :param tag: tag (or version) of the project.
         :param is_private: boolean value if the project should be visible just for user that creates it.
         :param pep_schema: assign PEP to a specific schema. [DefaultL: None]
```
Comment (nleroy917) [inline pepdbagent/modules/project.py:224]: typo? (`[DefaultL: None]`)

File: pepdbagent/modules/project.py [hunk 2]
```diff
@@ -315,6 +325,7 @@ def _overwrite(
         private: bool = False,
         pep_schema: str = None,
         description: str = "",
+        pop: bool = False,
```
Comment (nleroy917) [inline pepdbagent/modules/project.py:328]: I'm trying to think of any consequences that might occur from us defaulting to `False` here, but I can't immediately think of any...

File: pepdbagent/modules/project.py [hunk 3]
```diff
@@ -414,6 +426,7 @@ def update(
                     update_dict["subsamples"] = project_dict[SUBSAMPLE_RAW_LIST_KEY]
 
                 update_values = UpdateItems(**update_dict)
+                # update_values = UpdateItems(tag="ff")
```
Comment (nleroy917) [inline pepdbagent/modules/project.py:429]: Whats this `tag=ff`?

**Review by nleroy917 — APPROVED** [review]
Some questions, but approve.

File: pepdbagent/modules/project.py (hunk 3 above)
Comment (khoroshevskyi) [reply pepdbagent/modules/project.py:429]: ooops, I was testing, forgot to delete

File: pepdbagent/modules/project.py (hunk 2 above)
Comment (khoroshevskyi) [reply pepdbagent/modules/project.py:328]: I don't think it should cause any problems. pop is like private, but you can't change it. If you set it, it should be pop. So it's only metadata for pephub, nothing else. All the configurations will be then done in pephub (I am against this approach, but decision was made)

**Pull request merged**